internal/generator/fluentd/conf_test.go

```diff
@@ -202,7 +202,7 @@ var _ = Describe("Testing Complete Config Generation", func() {
   @type tail
   @id openshift-audit-input
   @label @INGRESS
-  path /var/log/oauth-apiserver/audit.log,/var/log/openshift-apiserver/audit.log
+  path /var/log/oauth-apiserver/audit.log,/var/log/openshift-apiserver/audit.log,/var/log/auth-server/audit.log
```

Comment:
it should be `/var/log/oauth-server/audit.log` as per Jira comment

Reply:
eagle eye